appserver/payara-appserver-modules/microprofile/healthcheck/src/main/java/fish/payara/microprofile/healthcheck/HealthCheckService.java

```diff
@@ -220,35 +196,50 @@ public void registerClassLoader(String appName, ClassLoader classloader) {
             }
         }
     }
+    
+    private Map<String, Set<HealthCheck>> getHealthChecks(HealthCheckType type) {
+        final Map<String, Set<HealthCheck>> healthChecks;
+        if (type == READINESS) {
+            healthChecks = readiness;
+        } else if (type == LIVENESS) {
+            healthChecks = liveness;
+        } else {
+            healthChecks = health;
+        }
+        return healthChecks;
+    }
+    
+        private Map<String, Set<HealthCheck>> getCollectiveHealthChecks(HealthCheckType type){
+        final Map<String, Set<HealthCheck>> healthChecks;
+        if (type == READINESS) {
+            healthChecks = readiness;
+        } else if (type == LIVENESS) {
+            healthChecks = liveness;
+        } else {
+            healthChecks = new HashMap<>(health);
```

Comment:
What I understand from this now is that when running health checks when doing type `READINESS` or `LIVENESS` we run just those but for `HEALTH` we run any kind of check? Is that it?